app/controllers/corrections/CorrectionCountryController.scala

```diff
@@ -86,15 +87,30 @@ class CorrectionCountryController @Inject()(
               updatedAnswers <- updateUserAnswers(value, periodIndex, countryIndex, correctionReturnPeriod)
               _ <- cc.sessionRepository.set(updatedAnswers)
               vatReturnResult <- vatReturnConnector.get(correctionReturnPeriod)
+              etmpVatReturnResult <- vatReturnConnector.getEtmpVatReturn(correctionReturnPeriod)
```

Comment:
If the API is not available (IE while toggled off), then this will cause the future to fail here and it will show an error page

Reply:
Would be good to add a test / verify to ensure it doesn't do the call when it's toggled off